step.yml

```diff
@@ -114,12 +111,7 @@ inputs:
       description: |-
         __Only available if `track` is `rollout`.__
 
-        Portion of the users who should get the staged rollout version of the app.
```

Comment:
From a user experience point, I feel that the predefined option list is a better option, as it would reduce errors due to misunderstanding the description.
The option to add a different option in the yml can be restored, so if a different number than in the list is needed can be provided by advanced users.
IMO add every tenth fraction from 0.1, 0.2, ... ,1.0

Reply:
Added inputs with 5%, please check.

Note: I have to do a dep update after the merge of https://github.com/bitrise-io/go-steputils/pull/14